services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/HankeServiceImpl.kt

```diff
@@ -181,8 +199,11 @@ open class HankeServiceImpl(private val hankeRepository: HankeRepository, privat
                     //   Do it below, outside this construction call.
                     hankeEntity.createdByUserId ?: "",
                     // From UTC without timezone info to UTC with timezone info
-                    if (hankeEntity.createdAt != null) ZonedDateTime.of(hankeEntity.createdAt, TZ_UTC) else null, hankeEntity.modifiedByUserId,
-                    if (hankeEntity.modifiedAt != null) ZonedDateTime.of(hankeEntity.modifiedAt, TZ_UTC) else null, hankeEntity.saveType
+                    if (hankeEntity.createdAt != null) ZonedDateTime.of(hankeEntity.createdAt, TZ_UTC) else null,
+                    hankeEntity.modifiedByUserId,
+                    if (hankeEntity.modifiedAt != null) ZonedDateTime.of(hankeEntity.modifiedAt, TZ_UTC) else null,
+
```

Comment:
Why extra row?

Reply:
To be consistent with the local style of one row = one parameter; the two params when in the end of the row (where they ended during one of Teemu's PRs, probably due to autoformatting) could be easily missed.

Reply:
Unless you referred to the empty line; which is just to group the audit-fields from other fields.